nautobot_golden_config/filters.py

```diff
@@ -284,3 +284,60 @@ class Meta:
 
         model = models.GoldenConfigSetting
         fields = ["id", "name", "slug", "weight", "backup_repository", "intended_repository", "jinja_repository"]
+
+
+class ConfigPlanFilterSet(BaseFilterSet, NameSlugSearchFilterSet):
+    """Inherits Base Class BaseFilterSet."""
+
+    q = django_filters.CharFilter(
+        method="search",
+        label="Search",
+    )
+    device_id = django_filters.ModelMultipleChoiceFilter(
+        queryset=Device.objects.all(),
+        label="Device ID",
+    )
+    device = django_filters.ModelMultipleChoiceFilter(
+        field_name="device__name",
+        queryset=Device.objects.all(),
+        to_field_name="name",
+        label="Device Name",
+    )
+    feature_id = django_filters.ModelMultipleChoiceFilter(
+        queryset=models.ComplianceFeature.objects.all(),
+        label="Feature ID",
+    )
+    feature = django_filters.ModelMultipleChoiceFilter(
```

Comment:
`compliance_feature` ?

Reply:
I would argue for consistency it should remain simply `feature`:

https://github.com/nautobot/nautobot-plugin-golden-config/blob/5b8eda117d0ae1d300e18491401d2634f7109031/nautobot_golden_config/models.py#L193-L196

Reply:
I'd agree that keeping it as `feature` for consistency is my choice here.